### PR TITLE
gives the developer feedback if something is wrong with the icons directory

### PR DESCRIPTION
**Pull request by deada92**
#359

### CODE REVIEW

**Comment by facebook-github-bot-0** [comment]
By analyzing the blame information on this pull request, we identified @drew-gross, @flovilmart and @Vortec4800 to be potential reviewers.

File: Parse-Dashboard/app.js [hunk 1]
```diff
@@ -26,10 +28,32 @@ function getMount(req) {
   return mountPath;
 }
 
+function checkIfIconsExistForApps(apps, iconsFolder) {
+  for (var i in apps) {
+    console.log(JSON.stringify(apps[i]));
```
Comment (drew-gross) [inline Parse-Dashboard/app.js]: Please remove extra `console.log`

Comment (deada92) [inline Parse-Dashboard/app.js]: Shame on me. Will remove it as soon as I'm home.

**Comment by drew-gross** [comment]
Awesome! The code looks good except for that one extra `console.log` in there. It also looks like there might have been a bunch of styling changes to the code, perhaps made by your editor automatically. Could you revert those changes? It's made it look like your PR changes pretty much the entire file when most of the lines actually aren't changed.

**Comment by deada92** [comment]
Oh I see. Yeah this is atoms pretty code style 😞 Gotta check if I can disable that. Else I will just paste those lines by hand.

**Comment by facebook-github-bot-5** [comment]
@deada92 updated the pull request.

**Comment by deada92** [comment]
I think this is better now..

**Comment by facebook-github-bot-1** [comment]
@deada92 updated the pull request.

**Comment by drew-gross** [comment]
Sweet! Looks good to me!

**Comment by natanrolnik** [comment]
Great to see that the icons feature is being continued. Thanks, @deada92 !

**Comment by deada92** [comment]
Sure anytime. If there's something else I can help with tag me. I'm just not that good with react else I would make it possible to upload the icon directly.

**Pull request merged**